lesson_71/HW_L71/controllers/product.controllers.js

```diff
@@ -49,4 +51,17 @@ export const getProductVideoByName = (req, res, next) => {
 
 export const getProductPreviewByName = (req, res, next) => {
   retrieveProductPreview(req, res, next, handleResponseGetMedia(res, next));
+};
+
+export const fetchProducts = async (req, res) => {
+  res.status(200).json(await getProducts());
```

Comment:
from the code style perspective, it is better to assign the result of  `getProducts` call to variable and then send it